docs/kb/semgrep-ci/github-upload-findings-in-security-dashboard.md

```diff
@@ -35,35 +35,44 @@ Changing the repository's default workflow permissions changes the permissions f
 This job only requires `write` permissions for `security-events`.
 
 ```yml
+# Name of this GitHub Actions workflow.
```

Comment:
I'd suggest not adding these comments in this example - they're helpful in our default configs to explain what's occurring, but I don't think they add a lot for customizing the config, and they distract from the important elements that are different.